src/commcare_cloud/manage_commcare_cloud/install.py

```diff
@@ -25,20 +24,24 @@ def run(self, args, unknown_args):
         put_virtualenv_bin_on_the_path()
         if not os.path.exists(ANSIBLE_ROLES_PATH):
             os.makedirs(ANSIBLE_ROLES_PATH)
-        
+
         if not os.path.exists(ANSIBLE_COLLECTIONS_PATHS):
             os.makedirs(ANSIBLE_COLLECTIONS_PATHS)
 
         env['ANSIBLE_ROLES_PATH'] = ANSIBLE_ROLES_PATH
         env['ANSIBLE_COLLECTIONS_PATHS'] = ANSIBLE_COLLECTIONS_PATHS
-        cmd_roles_parts = ['ansible-galaxy', 'install', '-f', '-r', os.path.join(ANSIBLE_DIR, 'requirements.yml')]
-        cmd_collection_parts = ['ansible-galaxy', 'collection', 'install', '-f', '-r', os.path.join(ANSIBLE_DIR, 'requirements.yml')]
-        
+        requirements_yml = os.path.join(ANSIBLE_DIR, 'requirements.yml')
+        cmd_roles_parts = ['ansible-galaxy', 'install', '-f', '-r', requirements_yml]
+        cmd_collection_parts = ['ansible-galaxy', 'collection', 'install', '-f', '-r', requirements_yml]
+
         for cmd_parts in (cmd_roles_parts, cmd_collection_parts):
             cmd = ' '.join(shlex_quote(arg) for arg in cmd_parts)
             print_command(cmd)
-            p = subprocess.Popen(cmd, stdin=subprocess.PIPE, shell=True, env=env)
-            p.communicate()
-
-        puts(color_notice("To finish first-time installation, run `manage-commcare-cloud configure`".format()))
-        return p.returncode
+            try:
+                subprocess.check_output(cmd, shell=True, env=env)
+            except subprocess.CalledProcessError as err:
+                print("process exited with error: %s" % err.returncode)
```

Comment:
Warnings and errors are already printed to stderr, so they will be visible. Could also `print(err.output)`, although that will likely mean that the error is buried way above in the output. We can change it if this turns out to be a bad decision.